src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/search/crosswikisearch/CrossWikiSearchPageObject.java

```diff
@@ -78,10 +78,12 @@ public CrossWikiSearchPageObject searchFor(String term) {
     return this;
   }
 
-  public CrossWikiSearchPageObject navigateToWikiResults() {
-    wait.forElementVisible(otherCommunitiesLink);
-    scrollAndClick(otherCommunitiesLink);
-    return this;
+  public void searchForWiki(String searchUrl, String term) {
+    try {
+      getUrl(searchUrl + "/Special:Search?search=" + term);
+    } catch (TimeoutException e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)